scripts/generate-tokens-wrapper.js

```diff
@@ -134,13 +134,77 @@ const processTokens = (srcPath, tokensDir) => {
         return;
     }
 
-    fs.writeFileSync(
-        path.join(__dirname, '..', 'tools', 'styles', tokensDir, fileName),
-        css
-    );
+    try {
+        fs.writeFileSync(
+            path.join(__dirname, '..', 'tools', 'styles', tokensDir, fileName),
+            css
+        );
+    } catch (er) {}
 };
 
 const processPackages = async (srcPath, tokensDir, index) => {
+    if (tokensDir === 'tokens') {
+        const packagename = tokenPackages[index];
+
+        let componentLevelTokensPath = path.join(
```

Comment:
Question: instead of hardcoding to the node_modules path should this leverage require.resolve?

Reply:
Good catch!